packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1096,6 +1096,20 @@ This section contains some general information about claims, as well as detailed
 * [ID Token](#id-token)
 * [Refresh Token](#refresh-token)
 
+### Token lifetime
+
+When you are using the [Okta Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), the lifetime of the JWT tokens is hard-coded to the following values:
+
+* **ID token:** 60 minutes
+* **Access token:** 60 minutes
+* **Refresh token:** 90 days
+
+When you are using a [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), you can configure the lifetime of the JWT tokens:
+
+* **ID token:** At least five minutes, no more than 60 minutes
+* **Access tokens:** At least five minutes, no more than 24 hours (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)). See the [Authorization Servers API](/docs/reference/api/authorization-servers/#actions-object).
+* **Refresh tokens:** At least 10 minutes, maximum of five years (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)). See the [Authorization Servers API](/docs/reference/api/authorization-servers/#actions-object).
```

Comment:
The refresh token has lifetime and idle time(window), `At least 10 minutes, maximum of five years` is for the idle time only.